pkg/operator/controller/instance_controller.go

```diff
@@ -438,7 +446,7 @@ func (r *InstanceReconciler) createAccessControl(ctx context.Context, name, name
 			{
 				APIGroups: []string{""},
 				Resources: []string{"pods"},
-				Verbs:     []string{"get", "list", "watch"},
+				Verbs:     []string{"get", "delete", "list", "watch"},
```

Comment:
see earlier comment on pods vs deployment.
So far we did not need to delete Pods, only read access was needed for determining the pod's IP addresses.